data/config/command_modifiers.yaml

```diff
@@ -8,7 +8,7 @@ PARAMETER:
     PARAMETER DATA 32 32 INT MIN MAX 0 "Data value"
     PARAMETER VALUE 64 32 FLOAT 0 10.5 2.5
     PARAMETER LABEL 96 96 STRING "COSMOS" "The label to apply"
-    PARAMETER BLOCK 192 0 BLOCK '' "Block of binary data"
+    PARAMETER BLOCK 192 0 BLOCK 0x0 "Block of binary data"
```

Comment:
Quotes look weird but I just tested this and COSMOS does still send a block of zeros when you specify an empty string. @ryanatball do you have an opinion?

Reply:
Maybe this is a problem in CmdSender or other tool?    Quotes in the config files do not by themselves, cause quotes in the packets.

Reply:
I do see issues in CmdSender, but also when using the XTCE converter. `&quot` ends up in the XML files when "" is used for BLOCK parameters.